selenium_set_prefs.js

```diff
@@ -0,0 +1,174 @@
+// Mozilla User Preferences - Set By Selenium Python
+
+// DO NOT EDIT THIS FILE.
+//
+// If you make changes to this file while the application is running,
+// the changes will be overwritten when the application exits.
+//
+// To change a preference value, you can either:
+// - modify it via the UI (e.g. via about:config in the browser); or
+// - set it within a user.js file in your profile.
+// - set it using Selenium's webdriver.Firefox Options API
+
```

Comment:
ok so do you mind explaining how I would use this file? Would I delete prefs when I see fit? Or is this file just for reference of what prefs are set?

Reply:
This is just a reference